Fix:

error: implicit declaration of function 'read' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'close' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://sourceforge.net/p/xgalaga/bugs/5/
--- title.c.orig	2008-03-05 15:58:16.000000000 -0600
+++ title.c	2020-11-15 10:45:46.000000000 -0600
@@ -5,6 +5,9 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <sys/types.h>
+#include <sys/uio.h>
+#include <unistd.h>
 #ifdef HAVE_FCNTL_H
 # include <fcntl.h>
 #endif
